packages/squiggle-lang/src/rescript/ReducerProject/ReducerProject_ProjectItem_T.res

```diff
@@ -0,0 +1,36 @@
+module Parse = Reducer_Peggy_Parse
+module ExpressionT = Reducer_Expression_T
+module InternalExpressionValue = ReducerInterface_InternalExpressionValue
+open Reducer_ErrorValue
+
+type sourceArgumentType = string
+type sourceType = string
+type rawParseArgumentType = result<Parse.node, errorValue>
+type rawParseType = option<rawParseArgumentType>
+type expressionArgumentType = result<ExpressionT.t, errorValue>
+type expressionType = option<expressionArgumentType>
+type continuation = InternalExpressionValue.nameSpace
+type continuationArgumentType = InternalExpressionValue.nameSpace
+type continuationType = option<continuationArgumentType>
+type continuationResultType = option<result<continuationArgumentType, errorValue>>
+type bindingsArgumentType = InternalExpressionValue.nameSpace
+type bindingsType = option<bindingsArgumentType>
+type resultArgumentType = result<InternalExpressionValue.t, errorValue>
+type resultType = option<resultArgumentType>
```

Comment:
Should this be 
```rescript
result<option<InternalExpressionValue.t>, errorValue>
```

rather than
```rescript
option<result<InternalExpressionValue.t, errorValue>>
```
Unless I'm misunderstanding something, I'd imagine if the code block doesn't return anything, but still raises an error, I'd like to know what that error is rather than just returning "None"?